vms/platformvm/state/state.go

```diff
@@ -122,6 +124,8 @@ type State interface {
 	GetStatelessBlock(blockID ids.ID) (blocks.Block, choices.Status, error)
 	AddStatelessBlock(block blocks.Block, status choices.Status)
 
+	GetBlockID(height uint64) (ids.ID, error)
```

Comment:
Also, I am seeing `GetBlockIDAtHeight`. Should it rename it to `GetBlockIDAtHeight` to be consistent?

Reply:
Either name is fine with me, but I like the consistency

Reply:
I renamed everything to align with `GetBlockIDAtHeight`